folium/map.py

```diff
@@ -217,6 +217,34 @@ def __init__(self, tiles='OpenStreetMap', name=None,
         {% endmacro %}
         """)
 
+class FeatureGroup(TileLayer):
+    def __init__(self, name=None, overlay = True):
+        """Create a FeatureGroup layer ; you can put things in it and handle them as a
```

Comment:
I guess my comment did not make it in the first try! I have a flaky connection today :unamused: 

Please change `overlay = True` to `overlay=True`